tests/src/Context/ScreenshotContext.php

```diff
@@ -124,12 +165,23 @@ public function createScreenshot($file_name, $message, $ext = TRUE) {
       file_put_contents($file_name, $screenshot);
     }
     else {
-      if ($ext) {
-        $file_name .= '.html';
+      try {
+        if ($ext) {
+          $file_name .= '.html';
+        }
+        $html_data = $this->getSession()->getPage()->getContent();
+        file_put_contents($file_name, $html_data);
+      }
+      catch (DriverException $e) {
       }
-      $html_data = $this->getSession()->getPage()->getContent();
-      file_put_contents($file_name, $html_data);
     }
+
+    if ($this->artifactsS3Uri) {
+      $output = ["aws s3 cp $file_name {$this->artifactsS3Uri}/"];
+      exec($output[0], $output);
```

Comment:
What is this proprietary command? Where is this being passed the AWS region, secret and key to use?

Let's use the AWS SDK which is included in our vendor folder like we do in all our other code that interacts with AWS.